atom/browser/api/atom_api_cookies.cc

```diff
@@ -141,21 +141,6 @@ void ResolvePromiseWithCookies(util::Promise promise,
   promise.Resolve(cookie_list);
```

Comment:
Why not remove this helper and rely on `ResolvePromise` ? Only issue would be that reference has to be removed.

```
template <typename T>
static void ResolvePromise(Promise promise, T result);
```